eng/pipelines/libraries/enterprise/linux.yml

```diff
@@ -0,0 +1,97 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
```

Comment:
Do we want CI to trigger on every push to master?

Reply:
I might adjust that after discussions with @safern and other infra folks. The primary goal is for these tests to be run for validating PRs. So, I'm not sure what value the pipeline trigger would be for other pushes to master. Besides PRs, what other kinds of pushes to master are there now?

Reply:
For this PR, I will revise the .yml file for the pipeline to disable running on pushes to branches. We'll just let it run on PRs.

The pipeline can also be manually run of course.